src/core/js/device.js

```diff
@@ -4,10 +4,12 @@ define(function(require) {
     var Bowser = require('coreJS/libraries/bowser');
     var $window = $(window);
 
-    Adapt.device = {
-        touch: Modernizr.touch,
-        screenWidth: getScreenWidth()
-    };
+    Adapt.device = {};
+    Adapt.device['touch'] = Modernizr.touch;
+    Adapt.device['screenWidth'] = getScreenWidth();
+    Adapt.device['screenHeight'] = getScreenHeight();
+    Adapt.device['orientation'] = getScreenOrientation();
+    Adapt.device['aspectRatio'] = getScreenAspectRatio();
```

Comment:
Yes, to be able to reuse the functions. When called from the object, Adapt.device is not defined.